File: testing/highlight-samples/sample.rb
# This is a comment
require 'json'

class User
  attr_reader :name, :age

  def initialize(name, age)
    @name = name
    @age = 42
  end

  def greet
    "Hi, #{@name}!"
  end
end

active = true
nothing = nil
pi = 3.14

File: testing/highlight-samples/sample.ex
# This is a comment
defmodule User do
  @name "hello world"
  @age 42

  def greet(name) do
    "Hi, #{name}!"
  end

  defp helper(x) do
    x * 3.14
  end
end

active = true
nothing = nil
atom_val = :hello
